src/main/java/com/minecolonies/core/client/gui/AbstractModuleWindow.java

```diff
@@ -32,12 +33,19 @@ public abstract class AbstractModuleWindow extends AbstractWindowSkeleton implem
      * @param building   {@link AbstractBuildingView}.
      * @param res        the resource String.
      */
+
     public AbstractModuleWindow(final IBuildingView building, final String res)
     {
         super(res);
         this.buildingView = building;
+        placeWindowModuleViews(this, building, -20, this.mc);
+
+    }
+    // need mc as its own arg because it's protected in Pane
+    static void placeWindowModuleViews(AbstractWindowSkeleton window, final IBuildingView building, int xOffset, Minecraft mc) {
```

Comment:
Give methods the correct access modifier, if this is intended to be called by subclasses only, it should be protected